PhotometryScript.py

```diff
@@ -0,0 +1,164 @@
+        global ReferenceBackgroundLoc
+        ReferenceBackgroundMag = img[y,x]
+        ReferenceBackgroundLoc = [y,x]
+        return ReferenceBackgroundMag, ReferenceBackgroundLoc
+
+    elif event == cv2.EVENT_RBUTTONDOWN:
+        global ObjectMag
+        global ObjectLoc
+        ObjectMag = img[y,x]
+        ObjectLoc = [y,x]
+        return ObjectMag, ObjectLoc
+    
+    elif event == cv2.EVENT_RBUTTONUP:
+        global ObjectBackgroundMag
+        global ObjectBackgroundLoc
+        ObjectBackgroundMag = img[y,x]
+        ObjectBackgroundLoc = [y,x]
+        return ObjectBackgroundMag, ObjectBackgroundLoc
+
+####################################
+# argument use in order to identify picture in command line
+# open with "python /path/to/script.py --image /path/to/picture.jpg"
+# only works with JPG as of now
+####################################
+
+ap=argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True)
+args= vars(ap.parse_args())
+
+####################################
+# reading in the jpg, cloning a copy, and then converting that copy to grayscale
+# creating window for it to pop up in
+# activating mouse function from above
+# and ESC on the picture will exit it out
+####################################
+
+img = cv2.imread(args["image"])
+clone=img.copy()
+img = cv2.cvtColor(clone, cv2.COLOR_BGR2GRAY)
+cv2.namedWindow("window")
+cv2.setMouseCallback("window", click)
+cv2.imshow("window", img)
+cv2.waitKey(0)
+cv2.destroyAllWindows()
+
+####################################
+# This is a way to find the brightest area near the pixel you clicked
+# Take the distance to background in either direction, search grid for brightest point
+# sum all the values of the square area selected
+####################################
+
+#print(ReferenceStarLoc, ReferenceBackgroundLoc)
+ReferenceDistance = (np.absolute(np.subtract(ReferenceStarLoc, ReferenceBackgroundLoc)))
+ReferenceRadius = np.amax(ReferenceDistance)
+print('The radius of the reference star is', ReferenceRadius)
+
+ReferenceRange = np.arange(-ReferenceRadius,ReferenceRadius+1,1)
+XReferenceRange = ReferenceStarLoc[0] + ReferenceRange
+YReferenceRange = ReferenceStarLoc[1] + ReferenceRange
+
+FullReferenceRange=[]
+for i in range(0,2*ReferenceRadius+1):
+    ColumnShape = []
+    ColumnValues = ((np.full((2*ReferenceRadius+1), YReferenceRange[i])))
+    ColumnShape  = np.column_stack((XReferenceRange,ColumnValues))
+    FullReferenceRange.append(ColumnShape)
+
+FullReferenceRange = np.array(np.concatenate((np.array(FullReferenceRange))))
+#FullestRange looks gross, but this was the only way to get it in one long 121 by 2 matrix
+#But yes, it is gross
+
+ReferenceMagValues = []
+for i in range(0,ReferenceRadius**2):
+    xVal = FullReferenceRange[i,0]
+    yVal = FullReferenceRange[i,1]
+    ReferencePixelValue=img[xVal,yVal]
+    ReferenceMagValues.append(ReferencePixelValue)
+
+ReferenceMagValues = (np.array(ReferenceMagValues)).astype(np.int16)
```

Comment:
This is a lot of rewrites of the same variable without any explanations as to why...